packages/web-components/src/components/va-date-text-input/test/va-date-text-input.e2e.ts

```diff
@@ -0,0 +1,332 @@
+
+    const label = await page.find('va-date-text-input >>> legend');
+    expect(label.innerText).toContain('This is a label');
+  });
+
+  it('sets a default date', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<va-date-text-input value="1999-05-03" />');
+
+    const month = await page.find('va-date-text-input >>> .input-month');
+    const day = await page.find('va-date-text-input >>> .input-day');
+    const year = await page.find('va-date-text-input >>> .input-year');
+
+    expect(month.getAttribute('value')).toBe('05');
+    expect(day.getAttribute('value')).toBe('03');
+    expect(year.getAttribute('value')).toBe('1999');
+  });
+
+  it('updates date based select and input fields', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<va-date-text-input value="1999-05-03" name="test" />',
+    );
+
+    const date = await page.find('va-date-text-input');
+    const elementMonth = await page.find('va-date-text-input >>> .input-month');
+    const elementDay = await page.find('va-date-text-input >>> .input-day');
+    const elementYear = await page.find('va-date-text-input >>> .input-year');
+    const handleMonth = await page.$('pierce/[name="testMonth"]');
+    const handleDay = await page.$('pierce/[name="testDay"]');
+    const handleYear = await page.$('pierce/[name="testYear"]');
+
+    // Click three times to select all text in input
+    await handleMonth.click({ clickCount: 3 });
+    await handleMonth.press('0');
+    await handleMonth.press('7');
+    // Click three times to select all text in input
+    await handleDay.click({ clickCount: 3 });
+    await handleDay.press('2');
+    await handleDay.press('1');
+    // Click three times to select all text in input
+    await handleYear.click({ clickCount: 3 });
+    await handleYear.press('2');
+    await handleYear.press('0');
+    await handleYear.press('2');
+    await handleYear.press('2');
+    await page.waitForChanges();
+
+    expect(elementMonth.getAttribute('value')).toBe('07');
+    expect(elementDay.getAttribute('value')).toBe('21');
+    expect(elementYear.getAttribute('value')).toBe('2022');
+    expect(date.getAttribute('value')).toBe('2022-07-21');
+  });
+
+  it('year input only allows for 4 characters to be used', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<va-date-text-input name="test" />');
+
+    const elementYear = await page.find('va-date-text-input >>> .input-year');
+    const handleYear = await page.$('pierce/[name="testYear"]');
+    await handleYear.press('2');
+    await handleYear.press('0');
+    await handleYear.press('2');
+    await handleYear.press('2');
+    await handleYear.press('3');
+
+    await page.waitForChanges();
+    expect(elementYear.getAttribute('value')).toBe('2022');
+  });
+
+  it('day and month input only allows for 2 characters to be used', async () => {
```

Comment:
Is this test still valid? It looks like you're allowing 1 and 2 characters now (from the test on line 242)